pkg/services/object/put/slice.go

```diff
@@ -4,7 +4,7 @@ import (
 	"context"
 	"fmt"
 
-	"github.com/nspcc-dev/neofs-node/pkg/services/object_manager/transformer"
+	"github.com/nspcc-dev/neofs-node/pkg/services/object/internal"
```

Comment:
Usually, `internal/smth` is used, because `internal` itself doesn't mean a lot.

Reply:
It's strange enough we have a whole package just for this interface. To me it looks a lot like a part of `pkg/services/object/put`. But there it'd be internal. But then who needs an internal interface?

Reply:
https://github.com/nspcc-dev/neofs-node/pull/2671#discussion_r1420316138 describes where i saw this interface and why

part of it is used by the get service too. i wanted to see more usages but somehow we have so many `WriteChunk` and `Write(object.Object) error` (not the `oi.Writer`) so it does not look so useful sadly. but we can continue this refactor

Reply:
`get` doesn't need it. It has the thing it needs already.

Reply:
dont you like how `HeaderWriter` is included in the `Target` and how every object interface can be placed in a single common file? every header should be written with the same interface does not matter what service does it